package.json

```diff
@@ -11,6 +11,7 @@
     "url": "git@github.com:koopjs/koop-output-dcat-us-11.git"
   },
   "devDependencies": {
+    "@esri/arcgis-rest-feature-layer": "^3.2.1",
```

Comment:
@drspacemanphd this is what I was talking about. This package is declared as a dependency in the example app, but when I run `npm run dev` the script chokes on a typescript error, saying that an interface from this package is not defined. Adding it as a dev and a peer dependency solves the issue and shouldn't affect consumers.